src/main/resources/io/viash/runners/nextflow/VDSL3Helper.nf

```diff
@@ -202,7 +206,7 @@ def _vdsl3ProcessFactory(Map workflowArgs, Map meta, String rawScript) {
   def createParentStr = meta.config.allArguments
     .findAll { it.type == "file" && it.direction == "output" && it.create_parent }
     .collect { par -> 
-      "\${ args.containsKey(\"${par.plainName}\") ? \"mkdir_parent \\\"\" + (args[\"${par.plainName}\"] instanceof String ? args[\"${par.plainName}\"] : args[\"${par.plainName}\"].join('\" \"')) + \"\\\"\" : \"\" }"
+      "\${ args.containsKey(\"${par.plainName}\") ? \"mkdir_parent '\" + (args[\"${par.plainName}\"] instanceof String ? args[\"${par.plainName}\"] : args[\"${par.plainName}\"].join('\" \"')).replace(\"'\", \"'\\\"'\\\"'\") + \"'\" : \"\" }"
```

Comment:
↑ I really had to read this a couple of times to understand what's going on :dizzy_face: 

Since escapeText already exists in the location where `createParentStr` is injected, we can use it to simplify the code a _little bit_

```suggestion
      "\${ args.containsKey(\"${par.plainName}\") ? \"mkdir_parent '\" + simplifyText(args[\"${par.plainName}\"] instanceof String ? args[\"${par.plainName}\"] : args[\"${par.plainName}\"].join('\" \"')) + \"'\" : \"\" }"
```

Reply:
Actually, we can also add a helper argument similar to how `inputFileExports` is generated:

```suggestion
      def contents = "args[\"${par.plainName}\"] instanceof List ? args[\"${par.plainName}\"].join('\" \"') : args[\"${par.plainName}\"]"
      "\${ args.containsKey(\"${par.plainName}\") ? \"mkdir_parent '\" + simplifyText(${contents}) + \"'\" : \"\" }"
```

Reply:
done